.appveyor.yml

```diff
@@ -39,4 +39,4 @@ before_test:
 test_script:
     - cd C:\projects\php-cs-fixer
     - vendor\bin\phpunit
-    - php php-cs-fixer --diff --dry-run -v fix
+    - php php-cs-fixer --diff-format=udiff --dry-run -v fix
```

Comment:
this one is still old

Reply:
Using `SOME_VAR=value command` does not work on Windows so I kept using the option instead.

Reply:
what about 
```
- set PHP_CS_FIXER_FUTURE_MODE=1
- php php-cs-fixer --diff --dry-run -v fix
```
?

Reply:
Done.